package.json

```
@@ -49,6 +47,7 @@
     "jest": "yarn test:jest",
     "screenshots": "node scripts/screenshots.js --screenshot-dir ./screenshots --styleguide-url file://$(pwd)/build/react --kss-dir ./build/styleguide",
     "test": "yarn test:jest",
+    "test:update": "yarn test react/examples.spec.jsx -u",
```

```suggestion
    "test:update": "jest -u",
```
in order to update also components like AppsSection.spec.jsx.snap

---

non justement je souhaite ajouter une commande pour n'upgrade que les snapshots de examples.
Quand on travaille sur un nouveau composant, après un build ce n'est pas nécessaire de relancer TOUS les tests de cozy-ui, seulement ceux du composant ainsi que les snapshots de example si on l'a rajouté dedans. C'est ce cas là que j'aimerai couvrir.

Peut-être renommer alors `test:update` en `test:examples` ?

---

pas d'opinion là-dessus.

Cette commande est très spécifique, donc pas dérangeante. Et elle améliore ta productivité, donc go